### ISSUE TITLE
this.global.BASE_URL is undefined

### ISSUE THREAD

**Issue by awronski** [studbits/nuxt-jest-puppeteer#7]
Hi!

I tried to run basic example but I get en error:
```
  Index page
    × renders index page (1ms)

  ● Index page › renders index page

    Protocol error (Page.navigate): Cannot navigate to invalid URL undefined

      at Promise (node_modules/puppeteer/lib/Connection.js:185:56)
      at CDPSession.send (node_modules/puppeteer/lib/Connection.js:184:12)
      at navigate (node_modules/puppeteer/lib/Page.js:525:39)
      at Page.goto (node_modules/puppeteer/lib/Page.js:505:7)
      at exports.visitPage (node_modules/nuxt-jest-puppeteer/lib/helpers.js:6:14)
```

I checked the url and the problem is that the this.global.BASE_URL is undefined.

https://github.com/studbits/nuxt-jest-puppeteer/blob/aab3030700df58e59ecd60d441043daa291ad9e3/lib/environment.js#L16

**Comment by alidcastano** [comment]
Are you running via cli or programmatically? 

the BASE_URL is set here: https://github.com/studbits/nuxt-jest-puppeteer/blob/aab3030700df58e59ecd60d441043daa291ad9e3/lib/setup.js

**Comment by alidcastano** [comment]
if running programmatically you need to set BASE_URL yourself

**Comment by tony19** [comment]
I ran into this same issue, and fixed it in #2 (it should be in the next release). You could try it with:

    npm i github:tony19-contrib/nuxt-jest-puppeteer#fix-default-setup

**Comment by alidcastano** [comment]
Quickly made release for you guys - @0.3.8 (https://github.com/studbits/nuxt-jest-puppeteer/releases/tag/0.3.8)

Thanks @tony19!
